content/app/guides/multi-app-solution/_index.en.md

```diff
@@ -0,0 +1,64 @@
+---
+title: General approach for making a multi-app solution in Altinn
+linktitle: Multi-app solution
+description: Considerations and explanations of how to go about when creating a multi-app solution
+weight: 250
+aliases:
+
+- /app/multi-app-solution/
+
+---
+
+Before continue reading these guidelines, please consider if
+a multi-app solution is what you need to fulfill the purpose
+of your form(s).
+
+## What is a multi-app solution?
```

Comment:
before going into the nitty gritty details I would lay down some concepts. 

E.g. Before getting into the implementation details of a multi-app solution, the components the solution consists of should be defined. 
 	An "original" application: [insert descr]
 	A "receiving" appliation: [insert descr]
 	Instance: [insert descr]

This guide takes you through a multi-app solution consisting of two apps, but the concept can be extended to contain several "original" and "receiving" applications.

Reply:
Not a fan of the term "original app", maybe end-user interaction app (which is not a better term), or something that better describes what the purpose of the app is. "Receiving" app could also be reconsidered. No suggestions as of now, but happy to chat to see if we can find better terms.